docker/python2/Dockerfile

```diff
@@ -8,4 +8,4 @@ RUN apt-get update -y && \
     python-setuptools && \
     rm -rf /var/lib/apt/lists/* /var/cache/apt/archives/*
 
-RUN pip install cupy==4.0.0b4 chainer==4.0.0b4
+RUN pip install chainer[cuda]==4.0.0b4
```

Comment:
We're going to use wheel in Dockerfile of Chainer/CuPy, so could you please keep as it is for now?
https://github.com/cupy/cupy/issues/967

Reply:
Definitely, sorry for any inconvenience.